### ISSUE TITLE
Refer ivar in initialize

### ISSUE THREAD

**Issue by yhara** [yhara/shiika#76]
Found that this raises error.

```rb
class Base
  def initialize(i: Int)
    @a = i
    putd @a  #=> "ivar `@a\' was not found"
  end
end
```

**Comment by yhara** [comment]
`ctx.iivars` holds the ivars found so far in `initialize`, but current `convert_ivar_ref` does not look into it.

**Comment by yhara** [comment]
`convert_ivar_ref` should check `ctx.iivars` if `ctx.is_initializer` is true.

**Comment by yhara** [comment]
Fixed by #214
